php/Terminus/Models/Site.php

```diff
@@ -536,18 +536,18 @@ public function removeTag($tag, $org)
     public function serialize()
     {
         $data = [
-        'id'            => $this->id,
-        'name'          => $this->get('name'),
-        'label'         => $this->get('label'),
-        'created'       => date(Config::get('date_format'), $this->get('created')),
-        'framework'     => $this->get('framework'),
-        'organization'  => $this->get('organization'),
-        'service_level' => $this->get('service_level'),
-        'upstream'      => $this->upstream->serialize(),
-        'php_version'   => $this->get('php_version'),
-        'holder_type'   => $this->get('holder_type'),
-        'holder_id'     => $this->get('holder_id'),
-        'owner'         => $this->get('owner'),
+            'id'            => $this->id,
+            'name'          => $this->get('name'),
+            'label'         => $this->get('label'),
+            'created'       => date(Config::get('date_format'), $this->get('created')),
+            'framework'     => $this->get('framework'),
+            'organization'  => $this->get('organization'),
+            'service_level' => $this->get('service_level'),
+            'upstream'      => $this->upstream,
```

Comment:
If you explicitly cast this as a string, then 0.x will not misinterpret it.
`'upstream' => (string)$this->upstream,`